src/tranquilo_dev/benchmark_report/task_create_index_for_benchmark_reports.py

```diff
@@ -0,0 +1,26 @@
+import pytask
+import snakemd
+from tranquilo_dev.config import PLOT_CONFIG
+from tranquilo_dev.config import SPHINX
+from tranquilo_dev.config import SPHINX_PAGES_BLD
+
+
+@pytask.mark.depends_on(SPHINX_PAGES_BLD / f"{list(PLOT_CONFIG.keys())[0]}.md")
+@pytask.mark.produces(SPHINX / "index.md")
+def task_create_index():
+    doc = snakemd.new_doc()
+
+    doc.add_heading("Welcome to tranquilo-dev's benchmark reports!")
+    doc.add_paragraph("This is the index page of the benchmark reports.")
```

Comment:
Do we want to write anything special on the index page?

Reply:
Not for now.